test/api_server_unittest.cc

```diff
@@ -1,10 +1,87 @@
 #include "../src/api_server.h"
+#include "../src/configuration.h"
+#include "../src/store.h"
 #include "gtest/gtest.h"
 
-namespace {
+namespace google {
 
-TEST(ApiServerTest, BaseTest) {
-  EXPECT_EQ("", "");
+class ApiServerTest : public ::testing::Test {
+  protected:
+   static void CallDispatcher(
```

Comment:
The `MetadataApiServer::Dispatcher` class is public (an oversight, probably — there's no need for it to be — but for now it is accessible), so you can just create instances in the test.
You would need to wrap `HandleMonitoredResource` into an accessor function like this, because that one *is* private.